lib/rocket_chat/messages/room.rb

```diff
@@ -95,6 +203,11 @@ def room_option_hash(options)
         options.each { |key, value| new_hash[Util.camelize(key)] = value }
         new_hash
       end
+
+      def validate_attribute(attribute)
+        raise ArgumentError, "Unsettable attribute: #{attribute}" unless \
```

Comment:
I'd suggest using `"Unsettable attribute: #{attribute || 'nil'}"` instead as calling `set_attr` with no args would result in `raise ArgumentError, 'Unsettable attribute: '`